src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -186,18 +186,28 @@ export default class PhotoField extends React.Component {
         };
       }
     }
+
+    this.setState({ warningMessage: '' });
+  }
+
+  onZoomSliderChange = (e) => {
+    this.refs.cropper.zoomTo(e.target.value);
   }
```

Comment:
This was previously the `zoom` function. Updated the name to be more specific. Also removed most of the logic- `zoomTo` causes `onZoom` to trigger and the logic between these two functions seemed the same except for how they were deriving values from their parameters. @cehsu please double check this assumption.

Reply:
This looks great @rianfowler.